tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 3.5.3
-envlist = flake8,py{37,36,35,27}-ansible{28,27,26,devel}
+envlist = flake8,metadata-validation,py{37,36,35,27}-ansible{28,27,26,devel}
```

Comment:
Let's not change this at all, please.

Reply:
This is desired, we do not want to encourage developers to avoid testing the metadata when they do test code locally. Is listed the 2nd because is faster than running the unittests. The are no implications on CI.

Reply:
Let's not discuss it here. It's unrelated to fixing deployment as the PR title states.